src/main/kotlin/com/criptext/mail/scenes/signin/SignInSceneController.kt

```diff
@@ -179,6 +180,7 @@ class SignInSceneController(
     private fun onLinkAuth(result: SignInResult.LinkAuth) {
         when (result) {
             is SignInResult.LinkAuth.Success -> {
+                if(model.linkDeviceState is LinkDeviceState.Accepted) return
```

Comment:
explain me the case when this condition is true

Reply:
There is an isolated case, in which the worker that is doing the link status check returns, and the websocket response is a little late, the thing is, that at the same time, both the retry and the websocket try to change the holder, which caused a crash in fabric, this prevents that from happening even though it is an isolated case.